examples/src/main/python/hbase_inputformat.py

```diff
@@ -16,6 +16,7 @@
 #
 
 import sys
+import ast
 
 from pyspark import SparkContext
 
```

Comment:
If we're expanding the example to illustrate dealing with `Result` that contains multiple records, it would probably be a good idea to expand the sample data illustrated in the comments below to make use of that feature (i.e. have the sample data have multiple records (column families?).